packages/data-provider/src/repository/submittedRepository.ts

```diff
@@ -98,13 +112,14 @@ const repository = (dependencies: BaseDependencies) => {
 			organization: string,
 			paginationOptions: PaginationOptions,
 			filter?: SQL,
-		): Promise<SubmittedDataRepository[] | undefined> => {
+		): Promise<SubmittedDataResponse[] | undefined> => {
```

Comment:
is the undefined return necessary here?

Reply:
Removed `undefined` as it's not necessary. In case there's no records found querying the database using Drizzle function `findMany` it will return `[]`